### PR TITLE
Improve docs for Gradle

### PR DESCRIPTION
**Pull request by myhau**
## Task

Resolves: #197

## Description

Improve docs for Gradle

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -27,6 +27,8 @@ dependencies {
 
     implementation("com.squareup.tools.build:maven-archeologist:0.0.10")
 
+    implementation("org.virtuslab:pulumi-google-native-kotlin:0.28.0.1")
```
Comment (jplewa) [inline build.gradle.kts:30]: Why is this needed?

Comment (myhau) [reply build.gradle.kts]: Good catch. Committed by mistake. Removed!

**Review by jplewa — APPROVED** [review]
